Makefile

```diff
@@ -497,4 +498,10 @@ deploy-from-index-image:
 .PHONY: test-acceptance-with-bundle
 ## Run acceptance tests with the operator installed from a given index image and channel
 test-acceptance-with-bundle: deploy-from-index-image
-	$(Q)TEST_ACCEPTANCE_START_SBO=remote $(MAKE) test-acceptance
\ No newline at end of file
+	$(Q)TEST_ACCEPTANCE_START_SBO=remote $(MAKE) test-acceptance
+
+.PHONY: release-manifests
+## prepare file for Release
```

Comment:
Can you provide a better help message?

Reply:
Done https://github.com/redhat-developer/service-binding-operator/pull/806/commits/10c7cabca396b5e1f0435bb5f25f5115db807cda